docs/sources/configure-client/grafana-agent/ebpf/configuration/_index.md

```diff
@@ -0,0 +1,122 @@
+
+The `pyroscope.ebpf` component is used to collect application performance profiles via eBPF.
+
+![Pyroscope ebpf diagram](ebpf.png)
+
+The `pyroscope.ebpf` runs on the host machine and collects stack traces associated with a process running on the current host.
+
+Using the `targets` argument, you can specify which processes and containers to profile on the machine. The `targets` can be from discovery components such as `dicovery.kubernetes`, `discovery.docker` and `discovery.dockerswarm`.
+To relabel discovered targets and set your own labels you can use the `discovery.relabel` component. For more information, see [Components](/docs/agent/latest/flow/concepts/components/).
+
+The `forward_to` parameter should point to a `pyroscope.write` component to send the collected profiles to your Pyroscope Server or [Grafana Cloud](/products/cloud/).
+
+| Name                      | Type                     | Description                                                  | Default | Required |
+|---------------------------|--------------------------|--------------------------------------------------------------|---------|----------|
+| `targets`                 | `list(map(string))`      | List of targets to group profiles by container id            |         | yes      |
+| `forward_to`              | `list(ProfilesReceiver)` | List of receivers to send collected profiles to.             |         | yes      |
+| `collect_interval`        | `duration`               | How frequently to collect profiles                           | `15s`   | no       |
+| `sample_rate`             | `int`                    | How many times per second to collect profile samples         | 97      | no       |
+| `pid_cache_size`          | `int`                    | The size of the pid -> proc symbols table LRU cache          | 32      | no       |
+| `build_id_cache_size`     | `int`                    | The size of the elf file build id -> symbols table LRU cache | 64      | no       |
+| `same_file_cache_size`    | `int`                    | The size of the elf file -> symbols table LRU cache          | 8       | no       |
+| `container_id_cache_size` | `int`                    | The size of the pid -> container ID table LRU cache          | 1024    | no       |
+| `collect_user_profile`    | `bool`                   | A flag to enable/disable collection of userspace profiles    | true    | no       |
+| `collect_kernel_profile`  | `bool`                   | A flag to enable/disable collection of kernelspace profiles  | true    | no       |
+| `demangle`                | `string`                 | C++ demangle mode. Available options are: `none`, `simplified`, `templates`, `full` | `none` | no |
+
+## Supported languages
+
+The `pyroscope.ebpf` component supports the following languages:
+
+- Go
+- Rust
+- C/C++ with frame pointers enabled
+- Python
+
+## Sending data Grafana Cloud Profiles
```

Comment:
We also "Send" instead of "Sending" in headings (i.e, preference for imperative instead of gerunds).
```suggestion
## Send data Grafana Cloud Profiles
```